src/main.ts

```diff
@@ -525,6 +524,16 @@ export class Purchases {
     const localeToBeUsed = selectedLocale || defaultLocale;
 
     return new Promise((resolve, reject) => {
+      window.history.pushState({ checkoutOpen: true }, "");
+      const onClose = () => {
+        window.removeEventListener("popstate", onClose);
```

Comment:
so when pressing the back button on the browser we close our modal..

I have **not** implemented yet that for example if you are in the checkout page and press back, you go back to the e-mail page. Right now it will close the whole modal.